src/pages/api/index.md

```diff
@@ -441,477 +320,64 @@ See a sample XLIFF file.
-1. **401 Unauthorized**: The 401 unauthorized error status response code indicates that the request was not processed because the destination resource's authentication credentials were invalid. Reauthenticating with right credentials is recommended. 
+1. **401 Unauthorized**: The 401 unauthorized error status response code indicates that the request was not processed because the destination resource's authentication credentials were invalid. Reauthenticating with right credentials is recommended.
 1. **403 Forbidden**: The 403 Forbidden error status code indicates that the server has received the request but has refused to allow it. The client may have insufficient rights to a resource.
 1. **404 Not Found**: The 404 error is returned when the server can’t find the requested resource. It could be broken or dead link because of file movements or entering a non-existent task name.
 
 ### Understanding the Successful Response
+
 ```java
 {
-  "context": {
-    "headers": {},
-    "configuration": null,
-    "entity": {
-      "locale": "fr-FR",
-      "status": "CANCELLED",
-      "updatedDate": "2021-09-16T04:02:32.969Z"
-    },
-    "entityType": "com.ADOBE.gcsservices.dto.AssetLocaleBean",
-    "entityAnnotations": [],
-    "entityStream": {
-      "closed": false,
-      "committed": false
-    },
-    "lengthLong": -1,
-    "responseCookies": {},
-    "requestCookies": {},
-    "entityClass": "com.ADOBE.gcsservices.dto.AssetLocaleBean",
-    "mediaType": null,
-    "length": -1,
-    "location": null,
-    "language": null,
-    "lastModified": null,
-    "date": null,
-    "acceptableLanguages": [
-      "*"
-    ],
-    "committed": false,
-    "allowedMethods": [],
-    "stringHeaders": {},
-    "entityTag": null,
-    "links": [],
-    "acceptableMediaTypes": [
-      {
-        "type": "*",
-        "subtype": "*",
-        "parameters": {},
-        "quality": 1000,
-        "wildcardType": true,
-        "wildcardSubtype": true
-      }
-    ]
-  },
-  "status": 200,
-  "cookies": {},
-  "mediaType": null,
-  "entity": {
-    "locale": "fr-FR",
-    "status": "CANCELLED",
-    "updatedDate": "2021-09-16T04:02:32.969Z"
-  },
-  "length": -1,
-  "location": null,
-  "language": null,
-  "lastModified": null,
-  "date": null,
-  "metadata": {},
-  "statusInfo": "OK",
-  "allowedMethods": [],
-  "stringHeaders": {},
-  "entityTag": null,
-  "links": [],
-  "headers": {}
+	"locale": "ja-JP",
+	"status": "CANCELLED",
+	"updatedDate": "2023-12-13T07:23:27.643Z"
 }
 ```
 
-
 The key parameters in response are:
 
-- **Project name**: DemoProject
-- **Asset name**: UserTest.html
-- **Task name**: TestTask806\_2
-- **Locale**: fr-FR [the locale of the asset to be cancelled]
+- **Project Id**: The Id of the project under which tasks are created
+- **Asset name**: An asset for which you want to cancel the translation workflow
+- **Task Id**: Unique string that defines the task Id
```

Comment:
**taskId:** A unique id for the translation task.
Also, keep only relevant fields which are part of response.

Reply:
done